csharp-basics/exercises/Loops/Loops/Exercise5/Program.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace Exercise5
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            Console.WriteLine("Enter first word:");
+            string word1 = Console.ReadLine();
+            Console.WriteLine("Enter first word:");
+            string word2 = Console.ReadLine();
+            string dot = ".";
+            List<string> listOfDots = new List<string>();
+
+            for (int i = 0; i < 50; i++)
```

Comment:
The words will be separated by enough dots so that the total line length is 30. ---> {word1}{string.Join("", listOfDots)}{word2}?

Reply:
Izlaboju